packages/react/src/CTABanner/CTABanner.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {forwardRef, type Ref} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import styles from './CTABanner.module.css'
+import {Heading as PlainHeading} from '../Heading'
+import {Text} from '../Text'
+
+export type CTABannerProps = BaseProps<HTMLDivElement> & {
+  children: React.ReactNode | React.ReactNode[]
+  align?: 'start' | 'center'
+  hasBorder?: boolean
+  hasShadow?: boolean
+  hasBackground?: boolean
+}
+
+export const Root = forwardRef(
+  (
+    {
+      align = 'start',
+      hasBorder = false,
+      hasShadow = true,
+      hasBackground = true,
+      className,
+      children,
+      ...props
+    }: CTABannerProps,
+    ref: Ref<HTMLDivElement>
+  ) => {
+    return (
+      <section
+        ref={ref}
+        className={clsx(styles.CTABanner, align === 'center' && styles['CTABanner--center'])}
+        {...props}
+      >
+        {hasShadow && (
+          <>
+            <div className={clsx(styles['CTABanner--Shadow'], styles['CTABanner--RightShadow'])} />
+            <div className={clsx(styles['CTABanner--Shadow'], styles['CTABanner--LeftShadow'])} />
+          </>
+        )}
+        <div
+          className={clsx(
+            styles['CTABanner--content'],
+            hasBorder && styles['CTABanner--border'],
```

Comment:
Left a comment about this in the stylesheet, but assuming we're following the BEM approach for naming this should be probably be..

```suggestion
            hasBorder && styles['CTABanner-content--border'],
```